### PR TITLE
Feature/event page

### PR DESCRIPTION
**Pull request by Merteg**

### CODE REVIEW

File: client/src/components/EventPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  }
+
+  renderTimeString() {
+    const start_day = moment(this.state.start_date).format('MMMM D, YYYY');
+    if (!this.state.end_date) {
+      return <p className="flow-text"><Icon>date_range</Icon>{start_day}<Icon>access_time</Icon>{moment(this.state.start_date).format('hh:mm')}</p>
+    }
+
+    if (start_day === moment(this.state.end_date).format('MMMM D, YYYY')) {
+      return <p className="flow-text"><Icon>date_range</Icon>{start_day}<Icon>access_time</Icon>{moment(this.state.start_date).format('hh:mm') + " - " +
```
Comment (ykobuk) [inline client/src/components/EventPage/index.jsx]: In my opinion, it's a bit hard to read, maybe need write  every `<Icon>` like this ->

```
<Icon>
    some text
</Icon>
```

and on 58line

**Comment by Merteg** [comment]
Update identation

File: client/src/components/EventPage/EventMeta.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Icon, Col } from 'react-materialize';
+import SingleMeta from './SingleMeta';
+import { isEmpty } from '../../utils';
+import { Link } from 'react-router-dom';
+
+
+class EventMeta extends Component {
```
Comment (RayMet) [inline client/src/components/EventPage/EventMeta.jsx:8]: What is class EventMeta doing?

Comment (Merteg) [reply client/src/components/EventPage/EventMeta.jsx:8]: What if I will say, render Components?
This Component renders meta information for the event.

**Review by ykobuk — APPROVED** [review]

**Review by RayMet — APPROVED** [review]

**Pull request merged**